packages/web-reporter/writeReport.ts

```diff
@@ -1,11 +1,44 @@
 import fs from "fs";
+import { TestCaseResult } from "@perf-profiler/types";
+
+// We should have better management of the time interval over the repo
+const MEASURE_INTERVAL = 500;
+
+export const getMeasuresForTimeInterval = ({
+  duration,
+  skip,
+  results,
+}: {
+  duration: number | null;
+  skip: number;
+  results: TestCaseResult[];
+}): TestCaseResult[] => {
+  const firstMeasureIndex = skip / MEASURE_INTERVAL;
+
+  return duration
```

Comment:
We should be able to skip some frames even if duration is null.

For example, I want to skip the first 500ms but keep all the data after